java-squid/src/main/java/org/sonar/java/model/declaration/MethodTreeImpl.java

```diff
@@ -56,14 +57,18 @@
   @Nullable
   private TypeTree returnType;
   private IdentifierTree simpleName;
+  private final SyntaxToken openParenToken;
   private final FormalParametersListTreeImpl parameters;
+  private final SyntaxToken closeParenToken;
   @Nullable
   private final BlockTree block;
+  @Nullable
+  private SyntaxToken semicolonToken;
   private final List<TypeTree> throwsClauses;
   private final SyntaxToken defaultToken;
   private final ExpressionTree defaultValue;
 
-  //FIXME nullable if semantic analysis is not set. Should have a default value.
+  // FIXME nullable if semantic analysis is not set. Should have a default value.
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png)Take the required action to fix the issue indicated by this comment.[![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)